src/Mapbender/CoreBundle/Resources/public/mapbender.element.layertree.tree.js

```diff
@@ -599,7 +540,6 @@
         _removeChild: function(changed) {
             var self = this;
```

Comment:
Use of self is redundant

Reply:
Code not touched, same as before, avoiding mixing in too much refactoring with a behaviour fix.